src/sandstorm/web-session.capnp

```diff
@@ -76,7 +76,17 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
   # `clientStream` is the capability which will receive server -> client messages, while
   # serverStream represents client -> server.
 
-  # TODO(someday): Add WebDAV and CalDAV methods?
+  # WebDAV methods
+  propfind @7 (path :Text, content :PropfindContent, context :Context) -> Response;
+  proppatch @8 (path :Text, content :ProppatchContent, context :Context) -> Response;
+  mkcol @9 (path :Text, content :MkcolContent, context :Context) -> Response;
+  copy @10 (path :Text, context :Context) -> Response;
+  move @11 (path :Text, context :Context) -> Response;
+  lock @12 (path :Text, content :LockContent, context :Context) -> Response;
+  unlock @13 (path :Text, content :UnlockContent, context :Context) -> Response;
+  acl @14 (path :Text, content :AclContent, context :Context) -> Response;
+  report @15 (path :Text, content :ReportContent, context :Context) -> Response;
+  options @16 (path :Text, context :Context) -> Response;
```

Comment:
Would it be bad form for us to treat OPTIONS like a regular method (as it is now) in web-session.capnp but have the proxy only pick up the DAV header? Or is that what you're referring to?